doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/security-management-rest-api-supported-permissions-ref.adoc

```diff
@@ -0,0 +1,97 @@
+[id='security-management-rest-api-supported-permissions-ref_{context}']
+= Supported permissions in {CENTRAL}
+
+The following are available permissions in {PRODUCT}. Administrators use these permissions to allow specific actions to a group, role, or user in {CENTRAL}.
+
+Priority::
+Priority is an integer that defines the precedence of users who are assigned multiple roles or groups. The default value of priority for a new group is `-100`. In {CENTRAL}, you can set an integer value as a priority, which is resolved using the following rules:
++
+--
+.Priority value table
+[cols="50%,50%", frame="all", options="header"]
+|===
+|Integer value
+|Priority
+
+|Between -5 and 0
+|LOW
+
+|Less than -5
+|VERY LOW
+
+|Equals to 0
+|NORMAL
+
+|Between 0 and 5
+|HIGH
+
+|Greater than 5
+|VERY HIGH
+|===
+--
+
+Home Page::
+`homePage` indicates the default landing page for users.
+
+Workbench::
+Workbench consists of the following defined permissions:
++
+--
+[source,json]
+----
+{
+	"editDataObject": true,
+	"plannerAvailable": true,
+	"editGlobalPreferences": true,
+	"editProfilePreferences": true,
+	"accessDataTransfer": true,
+	"jarDownload": true,
+	"editGuidedDecisionTableColumns": true
+}
+----
+--
+
+Pages, Editor, Spaces, and Projects::
+The following are possible values for the permissions based on the resource type:
++
+--
+* `PAGES: read,create,update,delete`
+* `EDITOR: read`
+* `SPACES: read,update,delete,create`
```

Comment:
For SPACES, suggest ordering as "read, create, update, delete" for consistency. However, that said, the industry standard follows the CRUD acronym for ordering, meaning "create, read, update, delete" (then build if applicable). So if the ordering is arbitrary, I recommend following the standard CRUD order, if it were me.